docs/getinvolved.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+title: "Get Involved"
+permalink: /getinvolved/
+---
+
+<h2>Introduction</h2>
+
+<p>The shadow-utils software collection is used for upstream maintenance of utilities used for managing account and shadown password files.</p>
+
+<hr/>
+
+<h2>Get involved!</h2>
+If you want to participate, you can contact us
+
+<ul>
+<li>on our <a href = "mailto:pkg-shadow-devel@alioth-lists.debian.net.">mailing list</a> ,</li>
+<li>on IRC (#shadow on freenode).</li>
+</ul>
+
+<p>Bug reports, patches, feature requests or any feedback are welcomed and can be created <a href="https://github.com/shadow-maint/shadow/issues">here</a>.</p>
+
+
+<hr/>
+<h2>Testsuite</h2>
+You can also contribute to the test suite and improve the coverage of these tests.
+
+The testsuite is currently executed on a Debian environment. You can help testing it on another environment.
+
+
+<h2>Authors and contributors</h2>
+
+{% for contributors in site.github.contributors %}
+  * [{{ contributors.login }}]({{contributors.contributors_url}})
+{% endfor %}
```

Comment:
These lines generate a list with the contributors with a link, but the links point to the home page of the project. Is this intentional?